### PR TITLE
Rename ReactiveOps to Fairwinds

### PR DESCRIPTION
**Pull request by rbren**
Remaining mentions: 
```
./cleanup.sh:kubectl delete crd rbacdefinitions.rbacmanager.reactiveops.io 2> /dev/null || true
./deploy/build.config:EXTERNAL_REGISTRY_BASE_DOMAIN=quay.io/reactiveops
./manifests/rbacdefinition.yaml:apiVersion: rbacmanager.reactiveops.io/v1beta1
./.circleci/config.yml:          username: "reactiveops+circleci"
```

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+  rok8s: fairwinds/rok8s-scripts@8.1.0
 
 workflows:
   build_and_push:
     jobs:
       - rok8s/docker_build_and_push:
           config_file: deploy/build.config
           docker-login: true
-          username: "fairwinds+circleci"
+          username: "reactiveops+circleci"
```
Comment (sudermanjr) [inline .circleci/config.yml]: This shouldn't be changed back.  I updated this repo already to use the new one.

File: create_cluster.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 gcloud beta container \
     clusters create cloud-build \
-    --project reactiveopsio \
+    --project fairwindsio \
```
Comment (sudermanjr) [inline create_cluster.sh]: This should probably actually change to just be something generic for the demo.

File: deploy/build.config [hunk 3]
```diff
@@ -1,4 +1,4 @@
 DOCKERFILE=('Dockerfile')
-EXTERNAL_REGISTRY_BASE_DOMAIN=quay.io/fairwinds
+EXTERNAL_REGISTRY_BASE_DOMAIN=quay.io/reactiveops
```
Comment (sudermanjr) [inline deploy/build.config]: I already updated, please don't change.

**Comment by rbren** [comment]
Thanks, fixed

**Review by sudermanjr — APPROVED** [review]

**Pull request merged**